src/helpers/issue.ts

```diff
@@ -186,17 +186,32 @@ export const getIssueDescription = async (issue_number: number): Promise<string>
       owner: payload.repository.owner.login,
       repo: payload.repository.name,
       issue_number: issue_number,
+      mediaType: {
+        format,
+      },
     });
 
     await checkRateLimitGit(response?.headers);
-    if (response.data.body) result = response.data.body;
+    if (response.data.body) {
+      switch (format) {
```

Comment:
Seems nice to have the capability but when would it be necessary to use anything but one of these formats?

Reply:
Sorry I missed this. I think it's good to have an option to choose the format if we need, no?

Reply:
I also missed this. Not sure but I'll resolve anyways.